Core/src/org/sleuthkit/autopsy/contentviewers/ApplicationContentViewer.form

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
It appears that the develop branch has been merged into the branch used for this pull request. We need the changes to the logging in the release-4.6.0 branch, but we need to keep the new application content viewer in the develop branch for eventual inclusion in the 4.7 release. Please address the additional review comments below and add the changed files to a release-4.6.0 branch and resubmit this pull request, thanks.